types/docx.d.ts

```
@@ -0,0 +1,21 @@
+
+  export class TextRun {
+    constructor(options: {
+      text: string;
+      bold?: boolean;
+      size?: number;
+    });
+  }
+
+  export class Packer {
+    static toBlob(doc: Document): Promise<Blob>;
+  }
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)